ajax/libs/polljs/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "polljs",
+  "filename": "poll.min.js",
+  "version": "0.3.0",
+  "description": "Javascript Timers and Intervals made easy.",
```

Comment:
"PollJS is a lightweight wrapper for the JavaScript Interval and Timer methods."
more clear